example/package.json

```diff
@@ -1,44 +1,27 @@
 {
+  "name": "example",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "expo start --dev-client",
-    "android": "react-native run-android",
-    "ios": "react-native run-ios",
+    "start": "expo start",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
     "web": "expo start --web",
-    "eject": "expo eject",
-    "postinstall": "patch-package"
+    "install-radar": "npm un --no-save react-native-radar ; npm i --no-save --install-links",
```

Comment:
I don't think --no-save works here without having react-native-radar in the dependencies, so just rm --no-save, and let dependencies be "react-native-radar": "file:.."

Reply:
builds fine, but doesn't resolve RNRadar after the app opens, might just be me though

Reply:
I don't think this is fine. The app building fine usually suggests that the native side stuff are in order. Usually js level issues are encountered at run time. I think changing the entry point to a path that does not exist is a bad idea.